.lando.yml

```diff
@@ -47,7 +47,7 @@ services:
       - composer install
       - ln -s /usr/local/quickstart-install-profile/.vscode /app/.vscode
   node:
-    type: node:18
+    type: node:20
     build:
```

Comment:
Blocked by https://github.com/lando/node/issues/66

Reply:
This worked for me with the latest lando release ([v3.21.0-beta.1](https://github.com/lando/lando/releases/tag/v3.21.0-beta.1))